content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -26,15 +28,24 @@ following cases:
 [Spring Boot starter]:
   https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.build-systems.starters
 
-The
+The OpenTelemetry Java agent has more automatic instrumentation features than
+the OpenTelemetry starter.
+
+You can use
```

Comment:
maybe add this to the end (other solutions)

Reply:
I would prefer to mention it at the beginning. In this way, the user could know the advantages and limitations of the OTel Java agent and the OTel Spring starter.